test/test_helper.rb

```diff
@@ -15,7 +15,9 @@ def access_sample_with_background(background, sample)
   end
 
   def sign_in(user)
-    @user = users(user)
-    post user_session_path, params: { 'user[email]' => @user.email, 'user[password]' => 'password' }
+    unless user.instance_of? User
```

Comment:
Is there a reason we can't require `sign_in` to be called in the same everywhere? Either

```
user = users(:admin_one)
sign_in(user)
```

or 

```
sign_in :admin_one
```

instead of allowing both?